tests/functional/u/undefined_variable.rc

```diff
@@ -0,0 +1,2 @@
+[basics]
+good-names=i,j,k,ex,Run,_,_dt
```

Comment:
Once again i think it is very risky to forbid using single character named variable...